### ISSUE TITLE
rpm pgadmin4-qt needs the python3-keyring package as dependency

### ISSUE THREAD

**Issue by wt** [pgadmin-org/pgadmin4#7220]
**Describe the bug**

Pgadmin4-qt will not start.

**To Reproduce**

1. Install Pgadmin4 as indicated on https://www.pgadmin.org/download/pgadmin-4-rpm/
2. Install pgadmin4-qt instead of pgadmin4-desktop (which doesn't seem to exist)
3. Run pgadmin4-qt
4. Get the error

**Expected behavior**

**Error message**

```
➜  postgresql pgadmin4-qt 
Traceback (most recent call last):
  File "/usr/lib/pgadmin4/pgAdmin4.py", line 49, in <module>
    import config
  File "/usr/lib/pgadmin4/config.py", line 923, in <module>
    from pgadmin.evaluate_config import evaluate_and_patch_config
  File "/usr/lib/pgadmin4/pgadmin/evaluate_config.py", line 12, in <module>
    import keyring
ModuleNotFoundError: No module named 'keyring'
```

**Workaround**

Install python3-keyring. After doing so, pgadmin4-qt works.

**Comment by yogeshmahajan-1903** [comment]
@wt 
Can you please provide you pgAdmin 4 version ? Also pgadmin can be started in 2 modes Desktop and server. Not sure what is getting installed with pgadmin4-qt.

**Comment by wt** [comment]
pgadmin4-qt installs the desktop version.

Version is 8.3.

This is about the rpm package not declaring a dependency.

**Comment by khushboovashi** [comment]
> pgadmin4-qt installs the desktop version.
We don't have anything as pgadmin4-qt. Can you please give the reference from where you are installing it? 
Screen-shot would be even better.
> 
> Version is 8.3.
> 
> This is about the rpm package not declaring a dependency.

**Comment by wt** [comment]
Okay, so I am running on Fedora Rawhide.

I just realized that I have no idea how I got pgadmin4 installed with this. I know that I started at this page:
https://www.pgadmin.org/download/pgadmin-4-rpm/

So, I regrouped and uninstalled and everything pgadmin4 related and removed the fedora repo rpm.

I installed the fedora repo rpm from the command on that page. After doing so I consistently get and error when dnf attempts to download new repomd.xml repo metadata from the pgadmin4 repo. It seems to be failing on accessing the file https://ftp.postgresql.org/pub/pgadmin/pgadmin4/yum/fedora/fedora-rawhide-x86_64/repodata/repomd.xml. This totally makes sense as the server doesn't have a directory for rawhide.

Okay, so where is the package coming form. Digging some more I found the following packages when searching for pgadmin4: pgadmin4, pgadmin4-qt, and a few others. 

I found that they exist directly in the fedora rawhide repository.

So, this is not a bug in the your distribution of pgadmin4.

Apologies for not finding this sooner.
